### DISCUSSION TITLE
local host issue

### DISCUSSION THREAD

**Discussion by Dmaster007** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#682] [General]
when i try to get code for final step at browser it shows localhost cant connect to server
![2022-03-22_10-50-05](https://user-images.githubusercontent.com/102041586/159413276-eb41a346-f3c2-4a7d-8384-5e4bb38b3fab.png)

**Comment by im-coder-lg** [comment]
It's this: https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/discussions/681#discussioncomment-2399388

Btw you should've just taken a Chrome screenshot without teams lol just common talk, that's your fix up there. Explained below:

The thing is that the code is _in the URL_.  It is just a query and this seems to have changed due to some recent commits. This was spotted by other people, not me, I am just forwarding the thread to the community.
@Melvin-Abraham are you adding a custom website to show "authenticated" which runs as a simple server via the app? Could be a swaggy addition here :thinking:

**Comment by abumuailek** [comment]
Same issue here. So what's the solution you think?

**Reply by im-coder-lg** [reply]
I think it's #671, there was a fix mentioned for this issue in that.

**Reply by im-coder-lg** [reply]
Al you have to do is just copy the code, I guess.
